src/nginx/azext_nginx/aaz/latest/nginx/deployment/certificate/_delete.py

```diff
@@ -57,14 +57,27 @@ def _build_arguments_schema(cls, *args, **kwargs):
             help="The name of targeted Nginx deployment",
             required=True,
             id_part="name",
+            fmt=AAZStrArgFormat(
+                pattern="^([a-z0-9A-Z][a-z0-9A-Z-]{0,28}[a-z0-9A-Z]|[a-z0-9A-Z])$",
```

Comment:
Is this some standardized format? Feels like another magic string that could be an issue in maintaining long term.

Reply:
This is from the latest stable swagger. Should this change, it will be reflected in future versions as well